probe/core/src/main/java/org/jboss/weld/probe/InvocationMonitor.java

```diff
@@ -86,7 +86,7 @@
 
     @Intercepted
     @Inject
-    private Bean<?> interceptedBean;
+    private transient Bean<?> interceptedBean;
```

Comment:
How can I know that the injected instance is always serializable? 
The `Bean` interface does not require this and the serializability (if that word exists) depends on the type parameter. Or not?